test/unit/util/option.js

```diff
@@ -109,4 +110,37 @@ describe('(libs/FP) Option type', () => {
       o.isEmpty().should.be.true();
     });
   });
+
+  describe('ramda interactions', () => {
+    describe('R.equals()', () => {
+      it('Option.none() should equal Option.none()', () => {
+        R.equals(Option.none(), Option.none()).should.be.true();
+      });
+
+      it('Option.of(...) should equal Option.of(...) the same value', () => {
+        R.equals(Option.of(1), Option.of(1)).should.be.true();
+      });
+
+      [
+        [ Option.none(), Option.of(0) ],
+        [ Option.of(''), Option.of(0) ],
+        [ Option.of(0),  Option.of(1) ], // eslint-disable-line no-multi-spaces
```

Comment:
Would you mind adding a case comparing an `Option` to a non-`Option` object? I know there's no way the test could pass, but since the logic in `'fantasy-land/equals'()` checks `instanceof`, I think it's worth testing.

```suggestion
        [ Option.of(0),  Option.of(1) ], // eslint-disable-line no-multi-spaces
        [ Option.of(1),  { value: 1 } ], // eslint-disable-line no-multi-spaces
```

Reply:
Done :+1: